constants.py

```diff
@@ -0,0 +1,15 @@
+BASE_URL = 'https://www.ebay.co.uk/'
+
+
+MAIN_PAGE_TITLE = 'Electronics, Cars, Fashion, Collectibles, Coupons and More | eBay'
+SEARCH_INPUT = 'Intel NUC'
```

Comment:
or as discussed change this to SEARCH_INPUT_INTEL and use it as in ```self.search_page.search_by_value(SEARCH_INPUT)``` and in the future if you want to search something else you add a new variable SEARCH_INPUT_MAC and use it like ```self.search_page.search_by_value(SEARCH_INPUT_MAC)```